Dockerfile

```diff
@@ -1,21 +1,31 @@
 # This dockerfile is specific to building Multus for OpenShift
-FROM openshift/origin-release:golang-1.10 as builder
-
-# Add everything
+FROM registry.access.redhat.com/ubi8 as rhel8
 ADD . /usr/src/plugins
-
 WORKDIR /usr/src/plugins
+RUN dnf install -y golang
```

Comment:
We prefer to pin golang releases - can you find a way to do this?

Reply:
So, IIUC, @jupierce and @sosiouxme will give us a rhel8 image which will have golang installed already. I'll revise this PR once that's done.